libs/dh/admin/feature-user-management/src/lib/tabs/roles-tab/dh-roles-tab.component.ts

```diff
@@ -111,12 +118,6 @@ export class DhUserRolesTabComponent {
   }
 
   readonly createUserRole = () => {
-    const url = this.router.createUrlTree([
-      dhAdminPath,
-      dhAdminUserManagementPath,
-      dhAdminUserRoleManagementCreatePath,
-    ]);
-
-    this.router.navigateByUrl(url);
+    console.log('create user role');
```

Comment:
Question: is this console.log intentional to be deployed?

Reply:
No, the method is not used anymore. Should be deleted now.